piccolo/apps/migrations/auto/migration_manager.py

```diff
@@ -788,4 +788,11 @@ async def run(self, backwards=False):
             await self._run_drop_columns(backwards=backwards)
             await self._run_drop_tables(backwards=backwards)
             await self._run_rename_columns(backwards=backwards)
+            # We can remove this for cockroach when resolved.
+            # https://github.com/cockroachdb/cockroach/issues/49351
+            # "ALTER COLUMN TYPE is not supported inside a transaction"
+            if engine.engine_type != "cockroach":
+                await self._run_alter_columns(backwards=backwards)
```

Comment:
Should we create a separate issue to track this, as a reminder to change it in the future?

Reply:
Tracked: https://github.com/piccolo-orm/piccolo/issues/632